core/src/test/java/com/gentics/mesh/core/s3binary/S3FileUploadClientTest.java

```diff
@@ -0,0 +1,78 @@
+package com.gentics.mesh.core.s3binary;
+
+import com.gentics.mesh.FieldUtil;
+import com.gentics.mesh.core.rest.node.NodeCreateRequest;
+import com.gentics.mesh.core.rest.node.NodeResponse;
+import com.gentics.mesh.core.rest.node.PublishStatusResponse;
+import com.gentics.mesh.core.rest.project.ProjectCreateRequest;
+import com.gentics.mesh.core.rest.project.ProjectResponse;
+import com.gentics.mesh.core.rest.schema.impl.SchemaResponse;
+import com.gentics.mesh.rest.client.MeshRestClient;
+import com.gentics.mesh.rest.client.MeshRestClientConfig;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Optional;
+
+public class S3FileUploadClientTest {
```

Comment:
Should be marked as `@Ignored` - this is not a CI-compatible test, but rather the test client.